commit 9b11d11ef63792bf3dae239d3e03df2027c85bcf (refs/changes/28/2428/2)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2017-08-21T11:43:07-04:00 (2 years, 2 months ago)
    
    PUBAPI-1426 sdc-cloudapi incorrectly handles machine create test failure
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/test/machines.70.test.js b/test/machines.70.test.js
index 238b6f9..3ca64cf 100644
--- a/test/machines.70.test.js
+++ b/test/machines.70.test.js
@@ -152,7 +152,7 @@ test('Wait For Running', function (t) {
 
 
 test('Get Machine,  with Firewall Enabled', function (t) {
-    if (MACHINE_UUID) {
+    if (!MACHINE_UUID) {
         return t.end();
     }
 
